clients/ui/bff/docs/contributing.md

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+## Running the linter locally
+The BFF directory uses golangci-lint to combine multiple linters for a more comprehensive linting process.
+
+You can use `brew` to install locally with:
+
+```shell
+brew install golangci-lint
+```
+
+and to run it simply use:
+
+```shell
+cd clients/ui/bff
+golangci-lint run
```

Comment:
Shall we add a target on makefile?

Similar to https://github.com/kubeflow/notebooks/blob/notebooks-v2/workspaces/backend/Makefile#L49-L56

Reply:
can do